src/i18n/nb_NO.ts

```diff
@@ -0,0 +1,1233 @@
+    .keys()
+    .forEach(key => (modules[key.split('/')[2]] = moduleRootFiles(key)));
+
+const t = {} as { [key: string]: unknown };
+
+furtherFiles
+    .keys()
+    .forEach(
+        key => (t[key.split('/')[1].replace(/\..*$/, '')] = furtherFiles(key))
+    );
+
+export default {
+    modules,
+    ...t,
+    error: {
+        title: 'LSS Manager: Error',
+        msg:
+            'Om denne feilen skjer ofte, vennligst rapporter det til LSSM teamet!',
+    },
+    warnings: {
+        version: {
+            title: 'Feil LSS Manager versjon',
+            text:
+                'Kjære bruker, vi ser at du ikke bruker siste versjon av LSS Manager. Den siste vesjonen er {version}, men du har {curver}. Last inn spillet på nytt med (Ctrl + F5, på Apple enheter Command + F5), dette bør fikse feilen. Om feilen fortsetter vennligst rapporter det til LSSM teamet.',
+            close: 'Lukk vindu og omlast spillet (anbefalt).',
+            abort: 'Likk vindu uten å omlaste spillet',
+        },
+    },
+    globalSettings: {
+        name: 'Innstillinger',
+        labelInMenu: {
+            title: 'Etikett istedenfor ikon i navigasjonslinjen',
+            description:
+                'Viser en enkel etikett i navigasjonslinjen i stedet for LSSM-logoen.',
+        },
+    },
+    vehicles: {
+        0: {
+            caption: 'Mannskapsbil',
+            color: '#cc0000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            wtank: 0,
+        },
+        1: {
+            caption: 'Lett mannskapsbil',
+            color: '#bb0000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+        },
+        2: {
+            caption: 'Stigebil',
+            color: '#d92626',
+            coins: 30,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 3 brannstasjoner.',
+        },
+        3: {
+            caption: 'Innsatslederbil brann',
+            color: '#d02525',
+            coins: 25,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 6 brannstasjoner.',
+        },
+        4: {
+            caption: 'Tungredningsbil',
+            color: '#ad1f1f',
+            coins: 25,
+            credits: 12_180,
+            minPersonnel: 1,
+            maxPersonnel: 4,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 4 brannstasjoner.',
+        },
+        5: {
+            caption: 'Ambulanse',
+            color: '#9c1c1c',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+        },
+        6: {
+            caption: 'Røykdykkerbil',
```

Comment:
The next after ambulance is the tankbil pls sort this https://www.nodsentralspillet.com/vehicle_graphics/4
And the other cars further down please